Process.c

```diff
@@ -52,6 +53,14 @@ void Process_setupColumnWidths() {
    assert(Process_pidDigits <= PROCESS_MAX_PID_DIGITS);
 }
 
+void Process_setUidColumnWidth(unsigned int maxUid) {
+   Process_uidDigits = ceil(log10(maxUid));
```

Comment:
I assume log10 would be expensive, so how about avoiding it when the UID is less than 5 digits?
```c
   if (maxUid < (unsigned int)pow(10, 5)) {
      Process_uidDigits = 5;
      return;
   }
```

Reply:
In that case you could even shortcut that calculation alltogether:

```c
   if (maxUid < 100000) {
      Process_uidDigits = 5;
      return;
   }
```

Reply:
@BenBE Actually the compiler will shortcut that for us, but I'm not sure we should define a constant `#define PROCESS_MIN_UID_DIGITS 5` for this purpose. Any opinion?

Reply:
If the `maxUid < pow(10, PROCESS_MIN_UID_DIGITS+1)` version is shortcutted by the compiler to produce effectively my suggestion from above, then I'd prefer that one, as it keeps that condition in sync with all the moving parts.

Reply:
It should be `maxUid < pow(10, PROCESS_MIN_UID_DIGITS)`. Because `pow(10, PROCESS_MIN_UID_DIGITS)` is 100000 and has 6 digits.

Reply:
Was still half asleep …

Reply:
Good point! Changed.